source/blog/2017-07-11-fedora-atomic-26-release.html.md

```diff
@@ -0,0 +1,551 @@
+---
+title: Fedora Atomic 26 Released
+author: jberkus
+date: 2017-07-11 18:00:00 UTC
+published: true
+comments: true
+tags: atomic, fedora, fedora 26
+---
+
+Fedora Atomic 26 is now generally available.  This contains updated package versions to match all of the content in Fedora Server 26, as well as updates to the container platforms.  While we release updates every 2 weeks, this release contains a collection of major improvements including:
+
+* latest rpm-ostree with improvements in package layering
+* default to Overlay2 filesystem for better container storage
+* Docker version 1.13.1
+* Latest versions of Cockpit and Atomic CLI
+
+We're all very excited about the steps forward Atomic Host is taking with this major release.  We hope you're just as excited to try them.  Read on for information about software, upgrading, and more.
+
+READMORE
+
+## Upgrading
+
+This is the first release for Fedora 26 Atomic Host coming only a week after the last release of Fedora 25 Atomic Host. Our next release will be in 3 weeks. Before you [upgrade your Fedora Atomic 25 machines](https://fedoraproject.org/wiki/Atomic_Host_upgrade) there are some things you should know.
+
+* Please make sure your root filesystem has a few gigabytes of free space. If your root filesystem never got extended to larger than 3GiB (the default on AWS), then you should resize it before attempting an upgrade.
+
+* If you use the Kubernetes software within Atomic Host then please wait until our next release of Atomic Host to upgrade. There is a [bug that is preventing the API server from starting](https://bugzilla.redhat.com/show_bug.cgi?id=1441218). If you don't rely on the built-in Kubernetes (e.g. you run containerized Kubernetes or OpenShift) this should not affect you. Before the next release we will publish a guide to upgrading your atomic host and making any configuration changes for the new version of kubernetes.
+
+* The storage driver now defaults to overlay2 instead of devicemapper for container storage.  This will not change existing installations.
```

Comment:
If you want to make the switch for in-place upgrades, see [this blog entry](http://www.projectatomic.io/blog/2017/05/migrate-fedora-atomic-host-to-overlay2/).